authentication.js

```diff
@@ -64,49 +66,78 @@ passport.use(samlStrategy);
 passport.use(new LocalStrategy(
     function(username, password, done) {
         // Find the user information in the database with the logged in values
-        client.get("ClassTranscribe::UserLookupTable::" + username, function(err, usr) {
-            // Display error if the account does not exist
-            if (!usr) {
-                var error = "Account does not exist";
-                console.log(error);
-                return done(null, false, { message: error });
-            } else {
-                var username = usr;
-                // Check if the user is verified their email address
-				client.hget("ClassTranscribe::Users::" + username, "verified", function(err, obj) {
-                    console.log("Is the email verified? " + obj);
-                    // Display error if email has not been verified
-					if (obj != "true") {
-						var error = "Email not verified";
-                        console.log(error);
-						return done(null, false, { message: error });
-					} else {
-						// Verify the inputted password is equivalent to the hashed password stored in the database
-						client.hget("ClassTranscribe::Users::" + username, "password", function(err, obj) {
-							var isCorrectPassword = passwordHash.verify(password, obj)
-                            console.log("Do the passwords match? " + isCorrectPassword);
-                            // Display error if password does not match the one stored in the database
-							if (!isCorrectPassword) {
-								var error = "Invalid password";
-                                console.log(error);
-								return done(null, false, { message: error });
-							} else {
-                                // Return the user if the login value matches the database
-								var suser = { firstname: usr['first_name'], lastname: usr['last_name'], email: username, verified: usr['verified'], university: usr['university'] };
-								return done(null, suser);
-							}
-						});
-					}
-				});
+        client.getUserByEmail(username).then(result=> {
+          // Display error if the account does not exist
+          if(!result) {
+            var error = 'Account does not exist';
+            console.log(error);
+            return done(null, false, {message : error});
+          } else {
+            var userInfo = result.dataValues;
+            // Check if the user has verified their email address
+            if (!userInfo.verified) {
+              var error = 'Email not verified';
+              console.log(error);
+              return done(null, false, {message : error});
             }
+            // Verify the inputted password is equivalent to the hashed password stored in the database
```

Comment:
Now that almost everything sql is working, 
Remove all commented out code for Redis from all files,
Including any require() statements of redis